### ISSUE TITLE
Losing input when not in focus

### ISSUE THREAD

**Issue by Asasiene23** [x360ce/x360ce#1253]
I am using the latest public version 4.17.15.0, while trying to emulate my Xbox One controller to steam as a Xbox 360 controller I noticed something interesting. At first the controller got recognized but didn't seem to send the proper instructions / or anything at all to steam, but after messing around I noticed that when i leave steam on my first Screen and focus on the Emulator setting window on my second screen the controller actually works. While keeping the focus on settings, the Joysticks work properly in the calibration window on steam (Specifically the big picture mode). After focusing on steam again the joysticks return to not working at all again.

Tldr: Xbox one controllers only work when the window of the emulator is in focus.

**Comment by davidmartos96** [comment]
Any update on this?

**Comment by nefarius** [comment]
This is a "feature" of the new driver stack from Microsoft used for Xbox **One** Controllers (in contrast to Xbox **360** Controllers) where the `xinputhid.sys` driver reacts to applications getting in and out of focus and suspends providing them with input updates once the "game" window (in this case the emulator) is not in focus anymore.

As per my current state of knowledge there is no way for x360ce to work around this the way the XInput subsystem is designed. I'll post again should I find something more of value.

**Comment by lupleime** [comment]
I have the same issue

**Comment by ozturkkl** [comment]
Any updates or fixes? I really need this to work...

**Comment by nefarius** [comment]
See my statement above, the app can't do much about this behaviour, send your kind words directly to Microsoft 😜

**Comment by davidmartos96** [comment]
> Any updates or fixes? I really need this to work...

What I ended up doing is disable the x360ce emulation for the One controller and just use it for my other "generic" controller

**Comment by EvoJocys** [comment]
This looks like it was addressed. From the changelog:

4.18.23 — Fixed: Polling rate setting had no effect.
4.19.17 — Fixed: Controllers were read slower while the window was open.

Please try 4.21: https://github.com/x360ce/x360ce/releases

Closing as part of a tracker clean-up after the 4.21 release. If it still happens on 4.21, reopen this issue with the version and what you see.
